test/benchmark/common/prepare_host.sh

```diff
@@ -43,5 +43,8 @@ prepare_libs() {
 prepare_fs() {
     # Disable unsupported ext2 features of Asterinas on Linux to ensure fairness
     mke2fs -F -O ^ext_attr -O ^resize_inode -O ^dir_index ${BENCHMARK_ROOT}/../build/ext2.img
-    make initramfs
+    # Check if ${BENCHMARK_ROOT}/../build/initramfs.cpio.gz exists
+    if [ ! -f "${BENCHMARK_ROOT}/../build/initramfs.cpio.gz" ]; then
```

Comment:
Remove such worthless comments. Reading the comment `# Check if ${BENCHMARK_ROOT}/../build/initramfs.cpio.gz exists` does not save people any time than doing the code `if [ ! -f "${BENCHMARK_ROOT}/../build/initramfs.cpio.gz" ]; then`.